### PR TITLE
Grade not undoing

### PR DESCRIPTION
**Pull request by waynewee**

### CODE REVIEW

File: src/main/java/igrad/model/module/ModuleCode.java [hunk 1]
```diff
@@ -11,9 +11,9 @@
 
 
     public static final String MESSAGE_CONSTRAINTS =
-        "Module code should contain two letters at the front and four "
+        "Module code should contain at two or three letters at the front and four "
```
Comment (yjskrs) [inline src/main/java/igrad/model/module/ModuleCode.java:14]: "two or three" ?

Comment (nathanaelseen) [reply src/main/java/igrad/model/module/ModuleCode.java:14]: think he missed out 'least' too

Comment (yjskrs) [reply src/main/java/igrad/model/module/ModuleCode.java:14]: "Module code should contain two or three letters at the front and four "

Comment (nathanaelseen) [reply src/main/java/igrad/model/module/ModuleCode.java:14]: oh ok sry!

Comment (nathanaelseen) [reply src/main/java/igrad/model/module/ModuleCode.java:14]: i think some  module codes have that

**Pull request merged**